starlark/src/stdlib/structs.rs

```diff
@@ -65,29 +64,25 @@ impl TypedValue for StarlarkStruct {
             .any(|x| x.same_as(other) || x.is_descendant(other))
     }
 
-    fn compare(&self, other: &Value, recursion: u32) -> Result<Ordering, ValueError> {
+    fn equals(&self, other: &Value, recursion: u32) -> Result<bool, TooManyRecursionLevelError> {
```

Comment:
Can we add some more equality tests in the structs test? I meant with struct of variable fields length.

Reply:
I'll do it.

Reply:
Sure.

Reply:
Tests for `struct` equality: #131.